internal/orchestrator/tasks/taskindexsnapshots.go

```diff
@@ -55,7 +55,7 @@ func indexSnapshotsHelper(ctx context.Context, st ScheduledTask, taskRunner Task
 	}
 
 	// collect all tracked snapshots for the plan.
-	snapshots, err := repo.Snapshots(ctx)
+	snapshots, err := repo.Snapshots(ctx, restic.WithFlags("--no-lock"))
 	if err != nil {
```

Comment:
Please don't pass this through, make it the default in the orchestrator. There probably isn't a scenario where we'd want to do this without --no-lock.